I've added the documentation now.
I was not able to build it, apparently due to PICMI-related stuff. The `requirements.txt` are missing at least the packages `typeguard sympy chevron jsonschema picmistandard`, after installing all them, I get the error:
```
Traceback (most recent call last):                                                                           
  File "/home/franz/virtualenvs/docs_picongpu/lib64/python3.10/site-packages/matplotlib/sphinxext/plot_directive.py", line 515, in _run_code                                                                                                                                                                                  
    exec(code, ns)                                                                                           
  File "<string>", line 25, in <module>                                                                      
  File "/usr/lib64/python3.10/importlib/__init__.py", line 126, in import_module                             
    return _bootstrap._gcd_import(name[level:], package, level)                                              
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import                                            
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load                                         
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked                                 
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed                               
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import                                            
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load                                                                                                                                                                                                                                                          
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked                                                                                                                                                                                                                                                 
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked                                                                                                                                                                                                                                                           
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module                                                                                                                                                                                                                                                     
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed                                                                                 
  File "/home/franz/git-repos/picongpu/lib/python/picongpu/utils.py", line 1, in <module>                                                                      
    raise ImportError(                                                                                                                                                                                                                                                                                                        
ImportError: module has been moved, please use: from picongpu.extra import utils 
```

So, it might be good if someone could check that the documentation compiles nicely.

--- a/docs/source/usage/plugins/openPMD_restart_config.json
+++ b/docs/source/usage/plugins/openPMD_restart_config.json
@@ -1,14 +1,14 @@
 {
-    "adios2": {
-      "dataset": {
-        "operators": [
-          {
-            "type": "blosc",
-            "parameters": {
-              "nthreads": 8
-            }
+  "adios2": {
+    "dataset": {
+      "operators": [
+        {
+          "type": "blosc",
+          "parameters": {
+            "nthreads": 8
           }
-        ]
-      }
+        }
+      ]
     }
   }
+}

